templates/includes/config.html

```diff
@@ -30,6 +30,16 @@
             // Wiki-specific settings will be placed here
             wiki: {
                 autosuggestTitleUrl: '{{ url('wiki.autosuggest_documents') }}'
+            },
+            // Just-in-time asset paths
+            assets: {
+                css: {
+                    "wiki-compat-tables": {{ css_bundle_urls("wiki-compat-tables") }}
+                },
+                js: {
+                    "syntax-prism": {{ js_bundle_urls("syntax-prism") }},
+                    "wiki-compat-tables": {{ js_bundle_urls("wiki-compat-tables") }}
+                }
```

Comment:
Just a small nitpick. We don't have strict rules about JavaScript coding conventions, but tend to use single quotes around strings.